src/engraving/dom/edit.cpp

```diff
@@ -6431,15 +6420,6 @@ void Score::undoAddCR(ChordRest* cr, Measure* measure, const Fraction& tick)
 
     Staff* ostaff = cr->staff();
     track_idx_t strack = ostaff->idx() * VOICES + cr->voice();
```

Comment:
Would `cr->track()` work here?  Just for clarity.

Reply:
Definitely, thanks